src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -452,7 +452,7 @@ def create_ave_image(ref_image: np.ndarray,
             'max_x': int(max_x)}
 
 
-def add_required_parameters(suite2p_args: dict):
+def add_modify_required_parameters(suite2p_args: dict):
     """Check that minimum parameters needed by suite2p registration are
```

Comment:
Should you also change the docstring to include the modify part that you added and the rationale for setting nonrigid = False?

Reply:
Not understanding your thumbsup here. You re-requested a review without making the change.

Reply:
Sorry, missed uploading it.